scripts/install.sh

```diff
@@ -196,14 +196,23 @@ info "Setting kube context to use namespace: $namespace"
 kubectl config set-context --current --namespace="$namespace"
 
 info "Getting ModelMesh Serving configs"
-if [[ -n $install_local_path ]]; then
-  if [[ -f $install_local_path ]] && [[ $install_local_path =~ \.t?gz$ ]]; then
-    tar -xf "$install_local_path"
-    cd "$(basename "$(basename $install_local_path .tgz)" .tar.gz)"
-  elif [[ -d $install_local_path ]]; then
-    cd "$install_local_path"
+if [[ -n $install_config_path ]]; then
+  if [[ -f $install_config_path ]] && [[ $install_config_path =~ \.t?gz$ ]]; then
+    tar -xf "$install_config_path"
+    cd "$(basename "$(basename $install_config_path .tgz)" .tar.gz)"
+  elif [[ $install_config_path =~ ^http.+ ]]; then
+    if [[ $install_config_path =~ \.t?gz$ ]]; then
+      curl -L $install_config_path -O
+      filename=$(basename $install_config_path)
+      tar -xf "$filename"
+      cd "$(basename "$(basename $filename .tgz)" .tar.gz)"
```

Comment:
Could you provide an example? I tried with `./scripts/install.sh -p https://github.com/kserve/modelmesh-serving/archive/refs/tags/v0.7.0.tar.gz -n modelmesh-serving` and got an error, "cd: v0.7.0: No such file or directory"

Reply:
I suppose the script can work with special built tgz and tar.gz files which contain just the config folder contents. It is not clear though based on the flag description, ".... Can be ModelMesh Serving tarfile or directory."

Reply:
Are you suggesting to replace the source tar.gz which is commonly for the entire project? Shall we name it more specifically like config-v0.8.0-rc0.tar.gz?

Reply:
Ah no, that will still remain with just a different download endpoint, but I can see how it might be confusing when they have the same names. So, I agree, let's name it more specifically.